### ISSUE TITLE
fails t/25proto.t without '.' in @INC ( do "t/proto1.p" failed, '.' is no longer in @INC )

### ISSUE THREAD

**Issue by kentfredric** [ingydotnet/inline-c-pm#61]
I have a templated answer I use for this on RT, but I don't have that template setup for github yet.

I assume you know the drill by now and the fix is obvious?

```perl
t/25proto.t .............. do "t/proto1.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto1.p"? at t/25proto.t line 8 (#1)                                              
    (D deprecated) Previously  do "somefile";  would search the current
    directory for the specified file.  Since perl v5.26.0, . has been
    removed from @INC by default, so this is no longer true.  To search the
    current directory (and only the current directory) you can write
     do "./somefile"; .

Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 12 (#2)
    (W uninitialized) An undefined value was used as if it were already
    defined.  It was interpreted as a "" or a 0, but maybe it was a mistake.
    To suppress this warning assign a defined value to your variables.

    To help you figure out what was undefined, perl will try to tell you
    the name of the variable (if any) that was undefined.  In some cases
    it cannot do this, so it also tells you what operation you used the
    undefined value in.  Note, however, that perl optimizes your program
    and the operation displayed in the warning may not necessarily appear
    literally in your program.  For example, "that $foo" is usually
    optimized into "that " . $foo, and the warning will refer to the
    concatenation (.) operator, even though there is no . in
    your program.


:
$@:
do "t/proto2.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto2.p"? at t/25proto.t line 16 (#1)
Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 20 (#2)

:
$@:
do "t/proto3.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto3.p"? at t/25proto.t line 24 (#1)
Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 28 (#2)

:
$@:
do "t/proto4.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto4.p"? at t/25proto.t line 32 (#1)
Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 36 (#2)

:
$@:
do "t/proto5.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto5.p"? at t/25proto.t line 40 (#1)
Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 44 (#2)

:
$@:
do "t/proto6.p" failed, '.' is no longer in @INC; did you mean do
        "./t/proto6.p"? at t/25proto.t line 48 (#1)
Use of uninitialized value $ret in concatenation (.) or string at t/25proto.t
        line 52 (#2)

:
$@:
t/25proto.t .............. Failed 6/6 subtests
```

**Comment by perlpunk** [comment]
Thanks, I created PR #65

**Comment by perlpunk** [comment]
Thanks, I created PR #65

**Comment by perlpunk** [comment]
@kentfredric we just released Inline::C 0.78
